resources/data/llama3-tokenizer/config.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Does this need to be here along with `resources/data/llama3-tokenizer/generation_config.json`?

Reply:
you're right, we don't actually use generation_config when resolving a tokenizer, we should remove this.